test/lit.cfg

```diff
@@ -1452,6 +1510,13 @@ if platform.system() != 'Darwin' or swift_test_mode == 'optimize_none_with_impli
 platform_module_dir = make_path(test_resource_dir, config.target_sdk_name)
 if run_vendor != 'apple':
     platform_module_dir = make_path(platform_module_dir, run_cpu)
+
+platform_dylib_dir = platform_module_dir
+if run_os == 'maccatalyst' and config.darwin_maccatalyst_build_flavor == "ios-like":
+    # When using the ios-macabi triple, look for module files
+    # the the 'maccatalyst' compiler resource directory.
```

Comment:
Nit: "the the", probably needs to be "in the".

Reply:
Fixed.